lms/djangoapps/program_enrollments/rest_api/v1/views.py

```diff
@@ -763,14 +763,14 @@ def get_programs_user_is_course_staff_for(self, user, program_type_filter):
         This function would take a list of course runs the user is staff of, and then
         try to get the Masters program associated with each course_runs.
         """
-        program_list = []
+        program_dict = {}
```

Comment:
Nit: you could make this a `set()` instead, since you don't need the keys of your dict.

Reply:
I cannot do this. I tried. The error I got was: `TypeError: unhashable type: 'dict'`. I think what I have is easy enough to read